definitions/tools/homer_tag_directory.cwl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'Creating the homer tag directory'
+baseCommand: ["/bin/bash", "homer_tag_directory.sh"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "chrisamiller/homer"
+    - class: ResourceRequirement
+      ramMin: 32000
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'homer_tag_directory.sh'
+        entry: |
+                set -o pipefail
+                set -o errexit
+
+                name=$1
+                bam=$2
+                reference_fasta=$3  #assumes that there's a .genome file next to the reference .fa
+                genome_file = `basename $3 .fa`
+                if [[ -f "$genome_file" ]];
```

Comment:
1) `-s` might a better test to use here (check if the file exists and has non-zero size)
2) could be a little cleaner to use `if [[ ! -s "$genome_file" ]];then ...`  and then eliminate the first "does exist" block.  (just say if it doesn't exist, throw the error and die).

Reply:
Another thing to think about is that this will not work if we switch the reference input to type File